integration_tests/test_rpath.py

```diff
@@ -0,0 +1,43 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015 Canonical Ltd
```

Comment:
This was terrible at the very beginning of the year. I kept feeling like it was 2015, so it was fine to see it. Then in, say, February, I started catching files that contained 2015 and putting 2016 in there. Now we're like halfway through the year, and I've apparently started assuming that all the copyrights say 2016 already :stuck_out_tongue: .